src/client/components/raw-data-modal/raw-data-modal.tsx

```diff
@@ -0,0 +1,312 @@
+import {Manifest} from "../../../common/models/manifest/manifest";
```

Comment:
I know this was not you but WebStorm but please watch out with the auto generated imports they tend to be wild. This should go in line 7